homeassistant/components/google_assistant/__init__.py

```diff
@@ -93,7 +93,12 @@ def _check_report_state(data):
 async def async_setup(hass: HomeAssistant, yaml_config: Dict[str, Any]):
     """Activate Google Actions component."""
     config = yaml_config.get(DOMAIN, {})
-    google_config = async_register_http(hass, config)
+    google_config = GoogleConfig(hass, config)
```

Comment:
The reason it was kept separate is so that it's clear which code belongs to the manual set up .

Reply:
but async_setup() won't be run for cloud? so it's still separate. If this is run, there are more problems since they view still got registered with wrong config structure.